config/locales/en.yml

```diff
@@ -253,7 +253,7 @@ en:
       th:
         amount: Amount
         order_number: "Order #"
-        order_status: Order Status
+        order_status: Current Order Status
```

Comment:
I think `app/views/facility_account_orders/index.html.haml` may still need it (if so, blame me: 5592a21a)